### PR TITLE
Board pca10059

### PR DESCRIPTION
**Pull request by jpconstantineau**
Blink sketch was tested on all GPIOs and it worked fine for all except for P1.00.  Not sure if this my dongle or if it's an issue with all dongles.

### CODE REVIEW

File: variants/pca10059/variant.cpp [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  48, 49, 50, 51, 52, 53, 54, 55,
+  56, 57, 58, 59, 60, 61, 62, 63
+};
+
+
+void initVariant()
+{
+  // LED1 & LED2
+  pinMode(PIN_LED1, OUTPUT);
+  ledOn(PIN_LED1);
```
Comment (hathach) [inline variants/pca10059/variant.cpp]: should be led off when powering on

**Comment by jpconstantineau** [comment]
@ladyada I am fine with getting tagged for issues specific for this board.  They are pretty cheap at digikey and mouser (they are both out of stock right now) and that's why including them provides a good jumping-in point.  That is, when/if you can SWD flash the bootloader to it...

**Comment by jpconstantineau** [comment]
I updated the 2 conflicting files.
The readme had urls pointing to 404 pages. (nordic made changes to their website)
The boards.txt includes the dongle.

**Comment by ladyada** [comment]
thanks! @hathach is on lunar new year break, we'll check it out when he returns :)

File: variants/pca10059/variant.cpp [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  40, 41, 42, 43, 44, 45, 46, 47,
+  48, 49, 50, 51, 52, 53, 54, 55,
+  56, 57, 58, 59, 60, 61, 62, 63
+};
+
+
+void initVariant()
+{
+  // LED1 & LED2
+  pinMode(PIN_LED1, OUTPUT);
```
[the remark this answers is not in the document]
Comment (hathach) [reply variants/pca10059/variant.cpp]: please turn off all LEDs when init

File: boards.txt [hunk 3]
```diff
@@ -204,4 +204,51 @@ pca10056.menu.debug.l1.build.debug_flags=-DCFG_DEBUG=1 -Os
 pca10056.menu.debug.l2=Level 2 (Full Debug)
 pca10056.menu.debug.l2.build.debug_flags=-DCFG_DEBUG=2 -Os
 pca10056.menu.debug.l3=Level 3 (Segger SystemView)
-pca10056.menu.debug.l3.build.debug_flags=-DCFG_DEBUG=3 -Os
+
```
Comment (hathach) [inline boards.txt]: You are probably remove the debug level 3 flag of pca10056 by accidence

**Comment by hathach** [comment]
closed due to lack of user interaction

**Pull request closed without merging**